src/Building/BuildingRepository.php

```diff
@@ -341,4 +341,70 @@ public function markBuildingWorkingStatus(int $userId, int $entityId, int $build
                 'status' => (int) $working,
             ])->execute();
     }
+
+    /**
+     * @return array<string[]>
```

Comment:
This goes for pretty much for all the repository methods added in this PR.

`string[]` indicates that the repository returns a list of strings e.g. `['one', 'two', 'three']`. To give the static analyzer the most possible information I would recommend specifying which keys are set in the array e.g. `array{name: string, cnt: string}`

Reply:
done. 
cadd473